twine/utils.py

```diff
@@ -99,23 +102,44 @@ def get_config(path: str = "~/.pypirc") -> Dict[str, RepositoryConfig]:
     return dict(config)
 
 
+def validate_url(repository_url: str) -> None:
```

Comment:
Thanks for wrapping this in a helper function; it makes `get_repository_from_config` easier to read.

Reply:
Agreed.

Reply:
Let's give this a more specific name, and indicate that it's not part of the public API of `twine.utils`:

```suggestion
def _validate_repository_url(url: str) -> None:
```

I think that makes the intent/scope clearer. If there's ever a need to validate more kinds of URL's, or we want a public method, we can revisit this.

Reply:
I have made it a private function in latest commit.